storybook/src/pages/amsterdam/FormPage/FormPage.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Alert,
+  Breadcrumb,
+  Button,
+  Column,
+  Field,
+  FieldSet,
+  FormFieldCharacterCounter,
+  Grid,
+  Heading,
+  Label,
+  Paragraph,
+  Radio,
+  Row,
+  Screen,
+  Select,
+  SkipLink,
+  TextArea,
+  TextInput,
+} from '@amsterdam/design-system-react'
+import { useState } from 'react'
+import { AppHeader } from '../common/AppHeader'
+
+export const FormPage = () => {
+  const [textareaLength, setTextareaLength] = useState(0)
+
+  return (
+    <>
+      <SkipLink href="#main">Direct naar inhoud</SkipLink>
+      <Screen maxWidth="wide">
+        <AppHeader />
+        <Grid paddingTop="small" paddingBottom="medium">
+          <Grid.Cell span={{ narrow: 4, medium: 6, wide: 8 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
+            <Breadcrumb>
+              <Breadcrumb.Link>Home</Breadcrumb.Link>
+            </Breadcrumb>
+            <Heading className="ams-mb--md">Contact</Heading>
+            <form className="ams-gap--md" id="main" onSubmit={(e) => e.preventDefault()}>
+              <Field>
+                <Label htmlFor="body">Wat wilt u aan de gemeente vragen?</Label>
+                <Paragraph id="bodyDescription" size="small">
+                  Een duidelijke beschrijving van uw vraag helpt ons bij het behandelen.
+                </Paragraph>
+                <TextArea
+                  aria-describedby="bodyDescription"
+                  id="body"
+                  onChange={(e) => setTextareaLength(e.target.value.length)}
+                  rows={4}
+                />
+                <FormFieldCharacterCounter length={textareaLength} maxLength={1000} />
+              </Field>
+              <FieldSet aria-describedby="contactDetailsDescription" legend="Wat zijn uw gegevens?">
```

Comment:
If your focus is on the first input the describedby property is not read.

Reply:
VoiceOver on my iPhone keeps reading out that description with every label or input I focus on. Seems like we need to test this further. In this PR or a new one? cc @alimpens

Reply:
Perhaps leave it out for now then?

Reply:
I think we should. It’s very important but makes the scope of this PR much larger. I added a [ticket](https://gemeente-amsterdam.atlassian.net/browse/DES-862) to our next sprint.